pkg/api/validation/schema_test.go

```diff
@@ -128,14 +128,14 @@ func NewFactory() (*Factory, error) {
 }
 
 func TestLoad(t *testing.T) {
-	_, err := loadSchemaForTest()
+	_, err := loadSwaggerSchemaForTest()
```

Comment:
test are failing and they are real failing. "loadSwaggerSchemaForTest" is not defined.